### ISSUE TITLE
ANDI Output on Domain Request Table

### ISSUE THREAD

**Issue by asaki222** [cisagov/manage.get.gov#3049]
### Issue description
The domain request tables fail accessibility compliance when no results are found. On normal search, the user is focused on the table itself. However, when no results are found, the table does not focus and the ANDI + voiceover output does not reflect that no results were found.

### Acceptance criteria
- [ ] On the domain request table, when using a keyboard to complete a search query (clicking the search button for instance), the user should be refocused on the table. 
- [ ] If the query has no results, focus should be brought to the table and the ANDI output should be "You are now in registered domains, no results found" (confirm with design).

### Additional context
More information about this ticket can be found in this [slack thread](https://cisa-corp.slack.com/archives/C07TZN9MQ9Z/p1730844958384119).

### Links to other issues
🔄 #2594 (originating issue)

**Comment by rachidatecs** [comment]
AC#2 should change to match what we're doing with the aria-live div provided by USWDS for this purpose, similar to what was done [here](https://github.com/cisagov/manage.get.gov/pull/3325/files#diff-e82274b35db049797605d5c259f2bb027f3a216f698e24ce2d22413c6f9d2beb)

**Comment by rachidatecs** [comment]
Had to do work to fix the members domain table
